spec/i18n_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+require 'i18n/tasks'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.